packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1192,4 +1293,192 @@ describe('TokenListController', () => {
       );
     });
   });
+
+  describe('executePoll', () => {
+    it('should call fetchTokenList with the correct chainId', async () => {
```

Comment:
so this is actually checking that the `fetchTokenList` function [here](https://github.com/MetaMask/core/blob/5e1044cd02180ad9bfe574a9ec4096b13ca0efbd/packages/assets-controllers/src/TokenListController.ts#L250) (which is confusingly the same name as the method in which it is called) is called with the correct arg.

Reply:
could rename the test `should call token service fetchTokenList function with the correct chainId` to avoid the confusion. Or rename the function to reduce confusion all around. Just didn't think that second option was properly scoped to this PR

Reply:
maybe we can import token-service without deconstruction `import TokenService from './token-service'` and also rename the test?

Reply:
eslint does not like your idea

<img width="717" alt="Screenshot 2023-10-06 at 4 29 31 PM" src="https://github.com/MetaMask/core/assets/34557516/50081ebb-6fc0-47bd-952d-789809434c58">

Reply:
renamed the function to `fetchTokenListByChainId`: https://github.com/MetaMask/core/pull/1763/commits/3ff7eee0c110ea95114839c9e86d734748f28847

Reply:
TokenService isn't exported as part of the asset-controller package. no changelog item necessary. nice!